.github/filters.yml

```diff
@@ -1,3 +1,5 @@
+mgmt:
+  - .github/workflows/**
```

Comment:
Should this also include other files in `.github` or are only the workflows considered part of the management stack? I'm thinking of things like the template files (issue, PR, and also the jinja templates).

Reply:
`mgmt` was a bad name for this field. This was specifically to prevent the CI CD workflow changes from being labelled as all of the stacks (because it appears in all of them).